src/ApplicationConfig.php

```diff
@@ -54,7 +54,7 @@ public function getIterator()
     }
 
     /**
-     * @return array
+     * @return array<integer|string>
```

Comment:
@tomphp 

Ha, interesting - I think I have seen this before somewhere, wasn't sure whether I should use it or not.

One thing, for consistency, should we use

```php
/**
 * @return array<int|string>
 */
```

instead?

Other than this, we are using the short forms for types.